### ISSUE TITLE
Reviewing OAuth permissions for ValyriaTear organisation

### ISSUE THREAD

**Issue by akien-mga** [ValyriaTear/ValyriaTear#575]
Hi @Bertram25 

Not directly related to the game and thus offtopic in this repo, but an issue is the simplest way to discuss it.

While giving permissions to GitLab to have access to my own account, I noticed that the ValyriaTear organisation is configured for unrestricted access, i.e. the fact that I gave GitLab.com access to my account also gave it access to ValyriaTear. See:

![2zngcw4pt](https://cloud.githubusercontent.com/assets/4701338/24859666/c656c6e2-1df1-11e7-87e5-98bb8d2dbc64.png)

I've restricted the access for OpenDungeons already, you might want to do the same for ValyriaTear at https://github.com/organizations/ValyriaTear/settings/oauth_application_policy

**Comment by akien-mga** [comment]
Note that once put in restricted mode, you can still grant access to the organisation to specific applications, as needed.

**Comment by Bertram25** [comment]
Hey @akien-mga 

Well, I've got this as third party authorized applications:
![image](https://cloud.githubusercontent.com/assets/765917/24876898/aef75110-1e2d-11e7-8dae-9d5a1c193e25.png)

And I've got no setup open OAuth policy:
![image](https://cloud.githubusercontent.com/assets/765917/24876958/e332a524-1e2d-11e7-8177-23990732bd80.png)

Thus, I'm wondering what makes you feel this is open. Maybe it's because you're member of the Valyria Tear organization?

I hope you're going well, btw.

Cheers!
